src/braket/circuits/synthesis/util.py

```diff
@@ -0,0 +1,281 @@
+# Copyright 2019-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import numpy as np
+from scipy.linalg import block_diag
+from typing import Tuple
+
+from braket.circuits.synthesis.predicates import is_diag, is_hermitian, commute
+
+
+def rx(theta):
+    """
+    Unitary for 1-qubit X rotations.
+
+    Args:
+        theta (float): the rotation angle.
+    """
+
+    # faster than expm(-0.5j * theta * x)
+    return np.array(
+        [
+            [np.cos(0.5 * theta), -1j * np.sin(0.5 * theta)],
+            [-1j * np.sin(0.5 * theta), np.cos(0.5 * theta)],
+        ]
+    )
+
+
+def ry(theta):
+    """
+    Unitary for 1-qubit Y rotations.
+
+    Args:
+        theta (float): the rotation angle.
+    """
+    return np.array(
+        [[np.cos(0.5 * theta), -np.sin(0.5 * theta)], [np.sin(0.5 * theta), np.cos(0.5 * theta)]]
+    )
+
+
+def rz(theta):
+    """
+    Unitary for 1-qubit Z rotations.
+
+    Args:
+        theta (float): the rotation angle.
+    """
+    return np.array([[np.exp(-0.5j * theta), 0], [0, np.exp(0.5j * theta)]])
+
+
+def to_su(u: np.ndarray) -> np.ndarray:
+    """
+    Given a unitary in U(N), return the
+    unitary in SU(N).
+
+    Args:
+        u (np.ndarray): The unitary in U(N).
+
+    Returns:
+        su (np.ndarray): The unitary in SU(N)
```

Comment:
```suggestion
        np.ndarray: The unitary in SU(N)
```
The return value doesn't have a name, just its type. Same goes for all other methods.